### PR TITLE
fix minimum time lag stripping, add prec to min time lag

### PR DESCRIPTION
**Pull request by rerpha**

### CODE REVIEW

File: correlator_pcaspy.py [hunk 1]
```diff
@@ -276,7 +276,9 @@ def take_data(self) -> None:
         no_repetitions = self.get_converted_pv_value(Records.REPETITIONS.name)
         wait_in_seconds = self.get_converted_pv_value(Records.WAIT.name)
         wait_at_start = self.get_converted_pv_value(Records.WAIT_AT_START.name)
-        min_time_lag = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
+        min_time_lag_unscaled = self.get_converted_pv_value(Records.MIN_TIME_LAG.name)
```
Comment (Tom-Willemsen) [inline correlator_pcaspy.py]: Would it be clearer to call this `min_time_lag_ns` or something like that?

File: correlator_pcaspy.py [hunk 2]
```diff
@@ -276,7 +276,9 @@ def take_data(self) -> None:
...
+        # Convert min time lag to seconds for comparison against lag data
+        min_time_lag = min_time_lag_unscaled / 1e9
```
Comment (Tom-Willemsen) [inline correlator_pcaspy.py]: magic number

```suggestion
        min_time_lag = min_time_lag_unscaled / NANOSECONDS_TO_SECONDS
```

**Review by Tom-Willemsen — CHANGES_REQUESTED** [review]
- This causes the existing IOC tests to fail
- Please add IOC tests covering the cases which are fixed by this PR.

**Comment by rerpha** [comment]
> * This causes the existing IOC tests to fail
> * Please add IOC tests covering the cases which are fixed by this PR.

thanks for the review, have addressed comments and added [this PR](https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/587) - i think the tests fixed cover the issue so didn't feel i needed to create another one

**Review by Tom-Willemsen — APPROVED** [review]

**Pull request merged**